src/NewTools-Debugger/StDebuggerMethodImplementorExtension.class.st

```diff
@@ -126,7 +131,7 @@ StDebuggerMNUExtension >> initializePresenters [
 		traits addAll: c allTraits.
 		classesAndTraits
 			addItemLabeled: c
-			do: [ self addProtocols: (c protocols)   ]
+			do: [ self addProtocols: ((c withAllSuperclasses reject:[:class| {Object. ProtoObject} includes: class]) flatCollect: #protocols) asSet   ]
```

Comment:
```Smalltalk
(c withAllSuperclasses reject:[:class| {Object. ProtoObject} includes: class])
```

can be replaced by:

```Smalltalk
(c withAllSuperclasses copyWithoutAll: {Object. ProtoObject})
```

I think this is easier to read, don't you think?

Side-note: I didn't know we could pass a selector as argument of `flatCollect:`

Reply:
Thanks!
I did the change, indeed this is simpler.